plugins/locator/stdloc/descriptions/global_stdloc.xml

```diff
@@ -71,30 +71,35 @@
 						<group name="GridSearch">
 							<description>
 								Find the source location by evaluating the hypocenter probability
-								of each grid cell and returning the maximum likelihood hypocenter.
-								The source time is derived from the weighted average of arrival
+								of each grid point and returning the maximum likelihood hypocenter.
+								The source time is derived from the weighted average of the arrival
 								travel times.
 								The solution can be further improved combining it with the Least Squares
-								algorithm, which will then be run for each grid cell, using the cell
-								centroid as initial location estimate. In this case only few big cells
-								are required.
+								algorithm, which will then be run for each grid point, using it as
+								initial location estimate. In this case only few sparse points are
+								required.
 							</description>
-							<parameter name="center" type="list:string" default="auto,auto,5">
-								<description>Latitude,longitude,depth[km]. </description>
+							<parameter name="center" type="list:string" unit="deg,deg,km" default="auto,auto,20">
+								<description>Grid center defined as: latitude,longitude,depth [km]. The
+								special value "auto" can be used and the corresponding latitude, longitude
+								 and/or depth will be automatically computed as the average of the arrival
+								station locations.
+								</description>
 							</parameter>
-							<parameter name="autoLatLon" type="boolean" default="true">
-								<description>If enabled, the grid center latitude and longitude
-								are automatically computed as the average of the arrival station
-								locations. The center depth should still be provided.
-							</description>
-							</parameter> 
-							<parameter name="size" type="list:string" unit="km" default="20,20,5">
-								<description>Grid X, Y, Z size in km</description>
+							<parameter name="size" type="list:string" unit="km" default="40,40,30">
+								<description>Grid size in km defined as: X,Y,Z direction extents around
+								the 'GridSearch.center', where X is the longitudinal extent, Y the
+								latitudinal extent and Z the vertical extent.
+								</description>
 							</parameter>
-							<parameter name="cellSize" type="list:string" unit="km" default="2.5,2.5,5">
-								<description>Cell X, Y, Z size in km</description>
+							<parameter name="numPoints" type="list:string" unit="km" default="">
```

Comment:
Is the unit really km? I guess this is a dimensionless integer.

Reply:
Good catch!